src/praisonai-agents/praisonaiagents/scheduler/config_store.py

```
@@ -327,6 +340,58 @@ def get_history(
+        if not state:
+            self.clear_state(job_id)
+            return
+        if not isinstance(state, dict):
+            logger.warning(
+                "Ignoring non-dict state for job '%s' (%s)", job_id, type(state).__name__,
+            )
+            return
+        try:
+            import yaml
+            size = len(yaml.safe_dump(state).encode("utf-8"))
+        except Exception as e:  # pragma: no cover - defensive
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)